src/App.js

```diff
@@ -1,13 +1,57 @@
 import React, { useState } from 'react';
+import {Switch, Route, NavLink} from 'react-router-dom';
 import './App.css';
+import Homepage from "./Pages/Homepage";
+import LoginPage from "./Pages/LoginPage";
+import Overview from "./Pages/Overview";
+import Blogpost from "./Pages/Blogpost";
+import {useHistory} from "react-router-dom";
```

Comment:
Het maakt tegnisch niet uit in welke volgorde je zaken importeerd maar ik zou jue adviseren daar wel een volgorde in aan te houden om het overzichtelijk te houden eerst de modules daarna de css en dan de componenten en dan ook weer de componenten gegroepeerd.
In dit geval zou ik dus op regel 2 import {Switch, Route, NavLink,useHistory} from 'react-router-dom'; en dan kan regel 8 weg